### DISCUSSION TITLE
Today i saw supabase is promoting paid course. How shameful that a campany doing back door things for making money that too from documentaion. Looks like that supabase policy is not reliable. They are in early stage , no campany would like to promote paid documentation course. You claim you are alternative of firebase , how many times you see firebase promoting paid course for documentation. They themselves put video, and prepare advance documentation for every aspect. "The day firebase launch postgress.... u guys are doomed"

### DISCUSSION THREAD

**Discussion by roker15** [supabase/supabase#3211] [Questions] [answered]
oday i saw supabase is promoting paid course. How shameful that a campany doing back door things for making money that too from documentaion. Looks like that supabase policy is not reliable. They are in early stage , no campany would like to promote paid documentation course. You claim you are alternative of firebase , how many times you see firebase promoting paid course for documentation. They themselves put video, and prepare advance documentation for every aspect. "The day firebase launch postgress.... u guys are doomed"

**Comment by roryw10** [comment, marked as answer]
Hi @roker15 

Thanks for flagging your concerns. I hope we are not doomed :) 

To clarify we don't make any money whatsoever from this course nor do we have a commercial relationship with the good folks over at Slip. Developers have a broad set of needs and different types of content appeal to their requirements. Making courses that go into this much depth isn't easy. Slip is making high quality content for developers and building a business around that and we are happy to highlight what they are doing.

We are not exclusively promoting content from Slip, or any single source for that matter. We take a proactive approach to promoting Supabase related content. We help amplify anything folks in the community are working on (check out our twitter feed for numerous examples of this [here](https://twitter.com/supabase/status/1432511957781917702) and [here](https://twitter.com/supabase/status/1427007690434351113)). We always ensure we attribute the creator and we try to promote a variety of content [for instance developer's own projects.](https://supabase.io/blog/2021/03/22/In-The-Loop)

Some folks give away content for free online and some folks charge for content, though this isn't the right forum to have a debate around free v.s paid content on the internet.

We are investing in creating our own content and guides to help users too - you can [check out our YouTube channel](https://www.youtube.com/channel/UCNTVzV1InxHV-YR0fSajqPQ) and [this video for example](https://www.youtube.com/watch?v=Ow_Uzedfohk&t=899s) or [our Auth deep dive series](https://www.youtube.com/watch?v=v3Exg5YpJvE&list=PL5S4mPUpp4OvE6MLbO1xAMvKLKT1sFsHF). We don't have the same resources that Google has so our videos might not be as well produced, we are committed to making content that enables our users though. We are prioritising the content we create based around community feedback so feel free to drop in anything you want us to cover.

**Reply by roker15** [reply]
You guys have great product. But Documentation is the key of how far you go. We are in era of self learning. People are reluctant of great products with poor documentation. Invest on documentation.

**Reply by zernonia** [reply]
@roker15 
> We are in era of self learning

Could you share what does this quotes mean? 

>  They themselves put video, and prepare advance documentation for every aspect

I wonder why there's so many Firebase course online, is it because they have poor documentation as well?
